index.html

```diff
@@ -16,49 +17,65 @@
 
 <body>
 
```

Comment:
Try wrapping your whole page in a container including the navigation
Give this container a `max-width: 900px`
Center the container using `margin: 0 auto`
And now proceed to remove the margins from your navigation and container home